libvirt/resource_libvirt_domain.go

```diff
@@ -996,3 +1048,56 @@ func getDomainInterfaces(domain *libvirt.VirDomain) ([]libvirt.VirDomainInterfac
 
 	return interfaces, nil
 }
+
+func getDomainInterfacesFromNetworks(domain defDomain, meta interface{}) ([]libvirt.VirDomainInterface, error) {
+
+	log.Printf("[DEBUG] getting domain addresses from networks")
+	var virDomain = make([]libvirt.VirDomainInterface, 0)
```

Comment:
In fact, I think you wouldn't need the `make` at all as you are `append`ing, right?